pages/reset-password.js

```diff
@@ -0,0 +1,36 @@
+import Image from 'next/image';
+
+import Background from '@/components/Background';
+import BackButton from '@/components/BackButton';
+import PasswordResetForm from '@/components/forms/PasswordReset';
+
+import logo from '@/public/logo.svg';
+
+export const passwordResetPageTestIds = {
+  headlineText: 'headline-text',
```

Comment:
Is this test-id too generic? Is there a risk that this one conflict with another headline-text somewhere? 
I would recommend to add the passwordReset information into it to reduce the risks of conflicts.

```suggestion
  headlineText: 'password-reset__headline-text',
```
(it looks like css BEM but when we think about it, it solves the same risk-for-conflict issue ^^)

Reply:
done in 62702641a4f14b18d1a77bfdc6c11f87694fc841